src/shared/modules/udc/udcDuck.js

```diff
@@ -247,6 +327,25 @@ export const bootEpic = (action$, store) => {
     })
 }
 
+const actionsOfInterest = [
+  PIN,
+  UNPIN,
+  REMOVE,
+  EXPAND,
+  ADD_FAVORITE,
+  LOAD_FAVORITES,
+  UPDATE_FAVORITE,
+  REMOVE_FAVORITE,
+  UPDATE
+]
```

Comment:
Moving to having more stats done like this would reduce maintenance burden but might generate a lot of events and not be as useful, since no data is sent.

Reply:
We might not care about all of these either

Reply:
I like this way of tracking as action types are defined by us and could never contain user data and all setup is done in one place without cluttering any other code